### ISSUE TITLE
Cannot perform validation on ReferenceInput values

### ISSUE THREAD

**Issue by ilaif** [marmelab/admin-on-rest#253]
I've tried applying a validation on a `<ReferenceInput/>` to be required as so:
```js
const validator = (value, values) => {
    if (!value)
        return ['Required field'];
}

<ReferenceInput source="resellerId" reference="reseller" allowEmpty={true} validation={validator}>
    <AutocompleteInput optionText="name" optionValue="id"/>
</ReferenceInput>
```
The value is undefined but the validation doesn't apply, although other inputs do (I tried also a global validation which did not work):
![image](https://cloud.githubusercontent.com/assets/3540825/22215645/ca33eb02-e1a4-11e6-95ca-fec81f97cfaf.png)

**Comment by ilaif** [comment]
@fzaninotto Just making sure you did not miss this issue. Thanks!

**Comment by fzaninotto** [comment]
I'm well aware of this, don't worry 😜

**Comment by fzaninotto** [comment]
Fixed by #289
